funding/manager.go

```diff
@@ -3164,9 +3165,19 @@ func (f *Manager) waitForTimeout(completeChan *channeldb.OpenChannel,
 
 	defer epochClient.Cancel()
 
+	// The value of waitBlocksForFundingConf is adjusted in a development
+	// environment to enhance test capabilities. Otherwise, it is set to
+	// DefaultMaxWaitNumBlocksFundingConf.
+	var waitBlocksForFundingConf uint32 = lncfg.
```

Comment:
nit/non-blocking: I know it's not super well-defined. But I think in this case breaking before the `=` character is preferred style.
Or alternatively:
```
	waitBlocksForFundingConf := uint32(
		lncfg.DefaultMaxWaitNumBlocksFundingConf,
	)
```